roiextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -19,6 +27,25 @@
     HAVE_NWB = False
 
 
+def nwb_metadata_recursive_update(metadata_base, metadata_input):
```

Comment:
ok but I would argue this function does not really do anything nwb-specific, so should not be called "nwb_"

Reply:
it take input nwb specific metadata though, since it is currently used only for that. 
I dont see it being useful any other places for now so IMO it would be better to have this name.

Reply:
this is a function for merging two dictionaries together.